test/e2e/storage/volume_provisioning.go

```diff
@@ -662,10 +662,12 @@ var _ = utils.SIGDescribe("Dynamic Provisioning", func() {
 				ClaimSize:    "2Gi",
 				ExpectedSize: "2Gi",
 			}
-
+			suffix := fmt.Sprintf("volume-lifecycle")
+			test.Class = newStorageClass(test, ns, suffix)
```

Comment:
What if we change the test to handle delayed binding instead?

Reply:
Added a commit to handle delayed binding.

From what I understand, this test should create its own SC anyway, so I left the first patch as well.

Reply:
Then we're not actually testing the default StorageClass, which I think is the purpose of this test.

Reply:
I assumed the opposite, since other tests explicitly mention they use the default SC. Dropped this code.